tests/test-data/test_configs/example.toml

```diff
@@ -88,8 +88,10 @@ zone_type = "Primary"
 ## file: this is relative to the directory above
 file = "example.com.zone"
 
-## if false, AXFRs requests will result in Refused responses
-# allow_axfr = false
+## if "deny", AXFRs requests will result in Refused responses
+## if "allow-signed", valid SIG(0) or TSIG authenticated AXFR requests are allowed.
+## if "allow-all", all AXFR requests (signed or unsigned) are allowed.
+axfr_policy = "deny"
```

Comment:
Hmm, if we're using snake_case for key names should we use snake_case for the values, too? Or did you find evidence that we're using kebab-case for other values in the configuration?